opentelemetry-zio-logging/src/main/scala/zio/telemetry/opentelemetry/zio/logging/TelemetryLogFormats.scala

```diff
@@ -0,0 +1,39 @@
+package zio.telemetry.opentelemetry.zio.logging
+
+import io.opentelemetry.api.trace.{Span, SpanContext}
+import io.opentelemetry.context.Context
+import zio.FiberRefs
+import zio.logging.LogFormat
+import zio.telemetry.opentelemetry.context.ContextStorage
+
+object TelemetryLogFormats {
```

Comment:
I think the better name would be `LogFormats`.

Reply:
Now I even think we might want to have a broader interface to be able to cover all parts of `zio-logging`.
For example, we can introduce `spanIdLabel` and `traceIdLabel` in addition to `spanId` and `traceId` (which should be renamed to `spanIdLogFormat` and `traceIdLogFormat` in such a case). So, instead of 
```
val spanIdLabel =  label("spanId", TelemetryLogFormats.spanId(ctxStorage))
```
a user would need to write:
```
val spanIdLabel = ZioLogging.spanIdLabel("spanId")
```
where `ZioLogging.spanIdLabel("spanId")` is defined as:
```
def spanIdLabel(label0: => String) = label(label0, spanIdLogFormat)
```

Reply:
I think we should create the same type of facade we have for the core `opentelemetry` module [OpenTelemetry.scala](https://github.com/zio/zio-telemetry/blob/series/2.x/opentelemetry/src/main/scala/zio/telemetry/opentelemetry/OpenTelemetry.scala). It may have only one `logFormats` ZLayer definition for now which provides an instance of `LogFormats` trait.

Reply:
Could or couldn't be named `ZioLogging`?

Reply:
"Could", sorry :). In other words - I suggest naming it `ZioLogging`.

Reply:
Fixed